dev-docs/RFCs/v5.0/viewport-animation-rfc.md

```diff
@@ -60,11 +60,23 @@ viewportAnimationDuration: (Number) Animation duration in milliseconds, default
 
 1. **animationDuration** {Number, default: 0} : Animation duration in milliseconds, default value disables animation.
 
-2. **easingFunction** {Function, default: t => t} : Easing function that can be used to achieve effects like "Ease-In-Cubic", "Ease-Out-Cubic", etc. Default value performs Linear easing. (list of sample easing functions: http://easings.net/)
+2. **animaitonEasingFunction** {Function, default: t => t} : Easing function that can be used to achieve effects like "Ease-In-Cubic", "Ease-Out-Cubic", etc. Default value performs Linear easing. (list of sample easing functions: http://easings.net/)
```

Comment:
Prop names are getting long, maybe just `animationEasing`?

Typo in `animaitonEasing`